### DISCUSSION TITLE
Refresh cache after editing tailwind.config

### DISCUSSION THREAD

**Discussion by jeremyzilar** [ben-rogerson/twin.macro#291] [Q&A]
Do you know if it possible to have the site rebuild and clear the cache after editing the `tailwind.config.js` file? 
To get the changes to the `tailwind.config` to take effect, I am finding that I need to 
- stop the server
- delete the `.next` folder
- restart the server

I am learning how to set up Next with Styled Components + Twin Macro + Typescript, and trying to figure out what part of the application is responsible for generating this.

**Comment by ben-rogerson** [comment]
The only fix I've found is to modify your start script:

```json
// package.json
"scripts": {
  "dev": "rimraf .next && next",
},
```

**Comment by 7iomka** [comment]
The same question :(

**Reply by ben-rogerson** [reply]
There's been no better solution on this yet, it's a difficult problem to solve.

**Comment by jeremyzilar** [comment]
This seems to mainly be an issue when setting up the foundation of a project, as I no longer experience the frustration because I am not editing that file as much. That said, I also took @ben-rogerson's advice and added it to my `package.json` scripts
```
"next:dev": "rm -rf .next && next",
```
